src/lifetimes.md

```diff
@@ -213,3 +218,82 @@ totally ok*, because it keeps us from spending all day explaining our program
 to the compiler. However it does mean that several programs that are totally
 correct with respect to Rust's *true* semantics are rejected because lifetimes
 are too dumb.
+
+
+
+# The area covered by a lifetime
+
+The lifetime (sometimes called a *borrow*) is *alive* from the place it is
+created to its last use. The borrowed thing needs to outlive only borrows that
+are alive. This looks simple, but there are few subtleties.
+
+This compiles, because after printing `x`, it is no longer needed, so it doesn't
```

Comment:
This "this" is quite far away from the example it refers to, so I was confused for a bit here. Could you move it closer, or else make the reference more explicit?

Reply:
I moved the paragraph below that to the very end, that might be a better place.